.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "ecmaVersion": 2018
+    "ecmaVersion": 2021
```

Comment:
Node 12.0.0 was released before that. Should we use an older version to make sure we don't use constructs that may not be supported?

Reply:
Some ES2020 things are supported by Node 12, because supported is often added before the versions finalize. This setting simply allows the parser to operate correctly on these features. If we want to disallow some, those should be in rules, not the parser. Anyway, tests should tell us if we're using unsupported features.